napari/components/layerlist.py

```diff
@@ -123,6 +124,36 @@ def _extent_world(self) -> np.ndarray:
         """
         return self._get_extent_world([layer.extent for layer in self])
 
+    def _get_min_and_max(self, mins_list, maxes_list):
```

Comment:
This function is not new code. It was split out from `_get_extent_world` for shared use by the new `_ranges` property.